source/code/source/playfab/PlayFabApiSettings.cpp

```diff
@@ -20,12 +20,24 @@ namespace PlayFab
     std::string PlayFabApiSettings::GetUrl(const std::string& urlPath) const
     {
         std::string fullUrl;
-        fullUrl.reserve(1000);
+        if(PlayFabSettings::connectionString == "" && connectionString == "")
+        {
+            fullUrl.reserve(1000);
 
-        fullUrl += "https://";
-        fullUrl += titleId;
-        fullUrl += baseServiceHost;
-        fullUrl += urlPath;
+            fullUrl += "https://";
+            fullUrl += titleId;
+            fullUrl += baseServiceHost;
+            fullUrl += urlPath;
+        }
+        else
+        {
+            if(PlayFabSettings::connectionString == "")
+            {
+                PlayFabSettings::connectionString = connectionString;
```

Comment:
I don't think we want to do this. My understanding is that PlayFabSettings is the static, program-wide configuration while PlayFabApiSettings is instanced and could potentially apply to a specific set of calls only. So, instead of setting the static variable when we get an instanced connection string, we should just treat it as an instanced override.

Reply:
Sure thing. I was trying to make it work with both. But since the static settings is considered legacy behavior, it would make a smaller change to apply across the remaining sdks that behave similarly.

Reply:
Updated to check instance first, then defer to static if present. Then build as we usually have.